website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,299 @@
+Namespaces are set via [Manager Options][ctrl-options] as described below. These Namespaces should be watched and 
+cached for the Client which is provided by the Manager.Only clients provided by cluster-scoped Managers are able 
+to manage cluster-scoped CRD's. For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching resources in all Namespaces (default)
+
+A [Manager][ctrl-manager] is initialized with no Namespace option specified, or `Namespace: ""` will 
+watch all Namespaces:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+...
+```
+
+## Watching resources in a single Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to a specific Namespace set the `Namespace` field 
+in [Options][ctrl-options]:
+
+```go
+...
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    Namespace:          "operator-namespace",
+    MetricsBindAddress: metricsAddr,
+})
+...
+``` 
+
+## Watching resources in a set of Namespaces
+
+It is possible to use [`MultiNamespacedCacheBuilder`][multi-namespaced-cache-builder] from 
+[Options][ctrl-options] to watch and manage resources in a set of Namespaces:
+
+```go
+...
+namespaces := []string{"foo", "bar"} // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+In the above example, a CR created in a Namespace not in the set passed to `Options` will not be reconciled by 
+its controller because the [Manager][ctrl-manager] does not manage that Namespace.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding Namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when managing a high number of Namespaces.
+
+## Restricting Roles and permissions
+
+An operator's scope defines its [Manager's][ctrl-manager] cache's scope but not the permissions to access the resources. 
+After updating the Manager's scope to be Namespaced, the cluster's [Role-Based Access Control (RBAC)][k8s-rbac] 
+permissions should be restricted accordingly.
+
+These permissions are found in the directory `config/rbac/`. The `ClusterRole` and `ClusterRoleBinding` manifests are 
+responsible for creating the permissions which allow the operator to have access to the resources.
+
+**NOTE** Only `ClusterRole` and `ClusterRoleBinding` manifests require changes to achieve this goal. 
+Ignore `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`, and files a name pattern of `auth_proxy_*.yaml`.
```

Comment:
- `Only ClusterRole and ClusterRoleBinding manifests require changes` is incorrect.
All manifests are ClusterRoles. We need to be specific about the manifest file names.
- Also mention why we ignore the other ones for now.
- Grammar nit.
```suggestion
**NOTE** For changing the operator's scope only the `role.yaml` and `role_binding.yaml` manifests need to be updated.
For the purposes of this doc, the other RBAC manifests `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`, and `auth_proxy_*.yaml` are not relevant to changing the operator's resource permissions.
```